src/fidesops/main.py

```diff
@@ -40,6 +42,75 @@
         allow_headers=["*"],
     )
 
+
+@app.middleware("http")
+async def dispatch_log_request(request: Request, call_next: Callable) -> Response:
+    fides_source: Optional[str] = request.headers.get("X-Fides-Source")
+    now: datetime = datetime.now(tz=timezone.utc)
+    endpoint = f"{request.method}: {request.url.path}"
```

Comment:
right now HTTP method gets filtered out by a validator in fideslog, but this will be fixed in https://github.com/ethyca/fideslog/pull/66